pkgs/by-name/cr/crates-tui/package.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  nix-update-script,
+}:
+let
+  version = "0.1.20";
+in
+rustPlatform.buildRustPackage {
+  pname = "crates-tui";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "ratatui";
+    repo = "crates-tui";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-K3ttXoSS4GZyHnqS95op8kmbAi4/KjKa0P6Nzqqpjyw=";
+  };
+
+  cargoHash = "sha256-ztLBM6CR2WMKR9cfBY95BvSaD05C+AEa6C/nOdDxqf0=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl ];
+
+  passthru.updateScript = nix-update-script { };
+
```

Comment:
Would be better to add https://nixos.org/manual/nixpkgs/unstable/#versioncheckhook